styles.css

```diff
@@ -158,7 +158,17 @@ body.dark-mode .expansion p {
   mix-blend-mode: multiply;
 }
 
+.go-mode-item > .item-image:not(.deselected),
+.go-mode-item:hover > .item-image {
+  outline: 2px solid black;
+  outline-offset: -2px;
```

Comment:
Check browser compatibility on `outline` CSS property

Reply:
Incompatible with Opera Mini and IE 11 browsers. Find alternatives or polyfills.